### ISSUE TITLE
Item in Html::radioButtonGroup is ignored

### ISSUE THREAD

**Issue by DarthLegiON** [kartik-v/yii2-helpers#38]
Hello!
I'm trying to use radioButtonGroup like this

``` php
echo Html::radioButtonGroup('status', $searchModel->status,
    [
        "" => 'All',
        1 => 'Correct',
        0 => 'Neutral',
        -1 => 'Incorrect'
    ],
    [
        'style' => 'white-space: nowrap',
        'encode' => false,
        'item' => function ($index, $label, $name, $checked, $value) {
        $radioButton = Html::radio($name, $checked, [
            'value' => $value,
        ]);
        return Html::label($radioButton . $label, null, [
            'class' => 'btn' . ($checked ? ' active' : '') . ($value > 0 ? ' btn-success' : ($value < 0 ? ' btn-danger' : ' btn-default')),
        ]);
        }
    ]
)
```

Closure named 'item' is ignored and even die() inside function does not stop execution.
Similar method in ActiveField class work fine. I think it's a bug because I found such code in its definition:
`if (!isset($options['item'])) {`
This code is missing in Html class and 'item' is rewritten and forgotten by application.

Please respond and/or fix this in your plugin. Thanks!

**Comment by DarthLegiON** [comment]
@kartik-v same issue is fixed in another plugin:
https://github.com/kartik-v/yii2-widget-activeform/issues/82
Please fix it here, thanks!

**Comment by kartik-v** [comment]
Resolved via 1a567d0.
